scripts/changeScanner/getChanges.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('shelljs');
+
+const getChanges = () => {
+  const execute = exec(`git diff --name-only latest ./packages`, {
+    silent: true,
+  }).stdout;
+
+  const changedFiles = execute.split('\n').filter(val => val);
```

Comment:
if you're checking for truthy things here you could do `.filter(Boolean);` instead

Reply:
Nice, never thought of that :D